.lighthouserc-1.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
           'assertions': baseConfig['basePerformanceAssertions']
         },
         {
-          'matchingUrlPattern': '^http://127.0.0.1:8181/blog-dashboard$',
+          'matchingUrlPattern': '^http://localhost:8181/blog-dashboard$',
```

Comment:
Will this change break the lighthouse test for python setup?

Reply:
no, because I changed the value for the `e2e PORT number` also from where it was taking its value to localhost. I made these changes after consulting it with @U8NWXD